simapp/simd/cmd/root.go

```diff
@@ -3,6 +3,7 @@ package cmd
 import (
 	rosettaCmd "cosmossdk.io/tools/rosetta/cmd"
 	"errors"
+	"github.com/cosmos/cosmos-sdk/types/mempool"
```

Comment:
Please move this import to the last block that has other github.com/cosmos/cosmos-sdk imports.

Reply:
sure!